example/App.js

```diff
@@ -39,7 +39,7 @@ export default function App() {
   const stringify = (obj) => JSON.stringify(obj, null, 2);
 
   useEffect(() => {
-    Radar.initialize("prj_test_pk_4ce462a534c0f749af3fdaac6fff9f485d65e720");
+    Radar.initialize("prj_test_pk_4ce462a534c0f749af3fdaac6fff9f485d65e720", true);
```

Comment:
Let's change this key back to `prj_test_pk_0000000000000000000000000000000000000000`. A test key accidentally made it here (now invalidated).